code/modules/clothing/suits/cloaks.dm

```diff
@@ -99,7 +99,7 @@
 	. += "<span class='notice'>You notice a powerful aura about this cloak, suggesting that only the truly experienced may wield it.</span>"
 
 /obj/item/clothing/neck/cloak/skill_reward/proc/check_wearable(mob/user)
-	return user.mind?.get_skill_level(associated_skill_path) < SKILL_LEVEL_LEGENDARY
+	return user?.mind?.get_skill_level(associated_skill_path) < SKILL_LEVEL_LEGENDARY
```

Comment:
This is called by `equipped()` and `attack_hand()`
Can either ever lack a `user` ? Do we need this sanity check?

Reply:
Similar to the other one, I can't think of a scenario where this will happen.

I think it's probably safer to approach it from the angle of - "Would we ever expect equipped() and attack_hand() to have no user?"

The answer to that is pretty much no. If this gets called without a user, I'm happy for it to runtime so it can be investigated and debugged. I'll remove the check.